test/OpenTelemetry.Contrib.Exporter.ApplicationInsights.Tests/OpenTelemetry.Contrib.Exporter.ApplicationInsights.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Do we target `net5.0` only or we want to cover more (e.g. `net461`)?

Reply:
Targeting 461 as well now